src/plugins/intel_gpu/src/kernel_selector/kernels/eltwise/eltwise_kernel_blocked_opt.cpp

```diff
@@ -422,6 +424,10 @@ static inline int SelectVecSizeFromFormat(const DataTensor& tensor) {
 static inline int GetInnerBatchBlockSize(const DataTensor& tensor) {
     auto layout = tensor.GetLayout();
     switch (layout) {
+    case DataLayout::bfyx:
+        if (tensor.LogicalSize() % 8 != 0) {
```

Comment:
Why 8 is used? Shouldn't it be vec_size instead?
And ideally we should reject unsupported cases in validate method, not during kernel's code generation

Reply:
Corrected now.  The check shouldn't use vec_size, should use the Feature().v.  Please review it.  

If compare with vec_size, it will fallback to eltwise_ref.  

In the unit tests, the comparison is correct if not using vec_size.  Use Feature().v similar check as spatial broadcasting.

Reply:
@clee30 , we should take into account the vec_size as well. Could you please try to add test like this:
`#define CASE_ELTWISE_TEST14  eltwise_mode::sum, {4, 7, 4, 4}, {1, 7, 1, 1}, format::b_fs_yx_fsv32, format::bfyx, "eltwise_blocked_opt"`
and check the generated code? It will probably use vload8 instruction for both inputs, which will lead to out of bound memory read in case of the second tensor

Reply:
@sshlyapn, yes, you are correct.    It will generate vload8.   The test is passing, and I guess just luck for out of bound read doesn't create some side effect.  

If I add vec_size check, all 14 tests will fallback to generic_eltwise_ref.   I'm working to add better check in validation to ensure with vec_size check and add additional test case to ensure eltwise_blocked_opt will be run.

Reply:
Done the change.